center_x: 0.250 # the center of the searching area
center_y: 0.0 # the center of the searching area
searching_area_radius: 0.2 # the radius of the searching area